src/components/learner-credit-management/styles/index.scss

```diff
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap');
+
+.font-family-roboto {
+  font-family: 'Roboto Mono', monospace;
+}
```

Comment:
Instead of defining this custom class name and importing the Roboto font, I would recommend relying on the `.text-monospace` to ensure these elements get the monospace font family. Grabbed from the [CSS utilities](https://paragon-openedx.netlify.app/foundations/css-utilities/) page on the Paragon documentation site.


<img width="912" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/42020779-1724-4883-9451-4e73a8ebccbb">